extract_endpoint/tests/test_post_to_endpoint.py

```diff
@@ -80,6 +80,11 @@ def sample_timestamp() -> str:
     return datetime.datetime(2013, 1, 1, 0, 0, 0).strftime("%Y-%m-%d %H:%M:%S")
 
 
+@pytest.fixture
+def sample_timestamp_filename() -> str:
+    return "timestamp.txt"
```

Comment:
You should probably either return `endpoint.TIMESTAMP_FILENAME` here or set `endpoint.TIMESTAMP_FILENAME` here. Otherwise the file used by the endpoint could differ from the one in this test file.